packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -367,9 +367,11 @@ export default class AiAssistantPanel extends Component<Signature> {
   }
 
   @action
-  private enterRoom(roomId: string) {
+  private enterRoom(roomId: string, hidePastSessions: boolean = true) {
```

Comment:
I thought typescript is smart enough to see the type from the default value

Reply:
Oh actually it is, just saw the lint error about this 😅

Reply:
ah didn't remove the type after deciding to set a default value, will do